google/cloud/spanner_v1/session.py

```diff
@@ -115,6 +124,8 @@ def create(self):
         metadata = _metadata_with_prefix(self._database.name)
 
         request = CreateSessionRequest(database=self._database.name)
+        if self._database and self._database.creator_role is not None:
```

Comment:
self._database should always be true for creating a session

Reply:
Made the change for this.